tests/spec/w3c/headers-spec.js

```diff
@@ -1141,7 +1141,7 @@ describe("W3C — Headers", () => {
       const otherLinks = [
         {
           class: "key-other-link",
-          key: "KEY",
+          key: "KEY:",
```

Comment:
It's a little "breaking change".

Reply:
It's a gray area (doesn't "break" anything) :) I searched around and not many specs that are affected. I'll send PRs for any I see, as most are not using this for any valid reason (e.g., they should use `"github"` instead).